pycryptobot.py

```diff
@@ -660,7 +657,7 @@ def executeJob(sc, app=PyCryptoBot(), state=AppState(), trading_data=pd.DataFram
                     else:
                         margin_text = '0%'
 
-                    output_text += ' | ' +  margin_text + ' (delta: ' + str(round(price - state.last_buy_price, 2)) + ')'
+                    output_text += ' | ' +  margin_text + ' (delta: ' + str(round(price - state.last_buy_price, 4)) + ')'
```

Comment:
what about using `precision` instead of `4` here?

Reply:
That makes sense :)